dashpay/src/main/kotlin/org/dashevo/dashpay/ContactRequest.kt

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Group
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package org.dashevo.dashpay
+
+import org.dashevo.dpp.document.Document
+import org.dashevo.platform.AbstractDocument
+
+class ContactRequest(document: Document) : AbstractDocument(document) {
+    val toUserId: ByteArray
+        get() = document.data["toUserId"] as ByteArray
+    val encryptedPublicKey: ByteArray
+        get() = document.data["encryptedPublicKey"] as ByteArray
+    val senderKeyIndex: Int
+        get() = document.data["senderKeyIndex"] as Int
+    val recipientKeyIndex: Int
+        get() = document.data["recipientKeyIndex"] as Int
+    val accountReference: Int
+        get() = document.data["accountReference"] as Int
+}
```

Comment:
This class is similar to DashPayContactRequest in dash-wallet.  it takes a document (normally from the network) and is assumed to be competely valid.  

this is using `get() =` syntax, although perhaps it would be better to simply assign the values?  If a document was invalid, then the object initialization would result in a crash, rather than the `get()` which would result in a crash when the item is needed.